examples/bri-3/src/bri/communication/agents/messaging.agent.spec.ts

```diff
@@ -128,6 +129,30 @@ describe('Messaging Agent', () => {
     );
     expect(resultDto.content).toEqual({ testProp: 'testValue' });
     expect(resultDto.signature).toEqual('xyz');
-    expect(resultDto.type).toEqual(0);
+    expect(resultDto.type).toEqual(BpiMessageType.Info);
+  });
+
+  it('Should return no errors and create message dto when validating correct JSON raw message of TRANSACTION type', () => {
+    // Arrange
+    const rawMessage =
+      '{ "id": "0a3dd67c-c031-4b50-95df-0bc5fc1c78b5", "fromBpiSubjectId": "71302cec-0a38-469a-a4e5-f58bdfc4ab32", "toBpiSubjectId": "76cdd901-d87d-4c87-b572-155afe45c128", "content": { "testProp":"testValue" }, "signature": "xyz", "type": 1}';
```

Comment:
Perhaps the enum type should be given string designations

Reply:
You mean for better readability? We can open a separate issue for that but i would not tackle it as a priority. What do you think?

Reply:
not  priority